mithril-common/src/entities/cardano_network.rs

```diff
@@ -74,6 +74,18 @@ impl CardanoNetwork {
             CardanoNetwork::TestNet(magic_id) => magic_id,
         }
     }
+
+    /// Evaluate the possibility of validating the request to allow unparsable block or not,
+    /// depending on the target Cardano network
+    pub fn allow_unparsable_block(&self, allow_unparsable_block: bool) -> StdResult<bool> {
+        let allow_unparsable_block = match self {
+            CardanoNetwork::MainNet => false,
+            CardanoNetwork::TestNet(id) if *id == PREPROD_MAGIC_ID => false,
```

Comment:
As it is blocked for `mainnet` and `preprod` networks, we should probably say that it "Will be ignored in (pre)production networks" in all the comments?